lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -97,6 +97,16 @@ public function GetQueueContents($ids, $order = 'createtime,desc', $state = NULL
 		elseif (empty($priority))
 			$priorityfilter = '';
 
+                if (isset($netdevids) && is_array($netdevids))
```

Comment:
Czy może wystąpić sytuacja w której $netdevids będzie array()-em nie zawierającym w ogóle elementów?

Reply:
To może dorzucić warunek && !is_null($netdevids) ?

Nie widzę takiej możliwości. W pozostałych warunkach jest tak samo.

Reply:
Poprawiłem we wszystkich warunkach.